module/.storybook/main.js

```diff
@@ -1,59 +1,27 @@
 module.exports = {
-  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
-  addons: ['@storybook/addon-links', '@storybook/addon-essentials', '@storybook/addon-a11y'],
+  stories: ['../../src/**/*.stories.mdx', '../../src/**/*.stories.@(js|jsx|ts|tsx)', '../../docs/**/*.stories.mdx'],
+  addons: [
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@storybook/addon-interactions',
+    '@storybook/addon-a11y',
+    '@storybook/test-runner',
+  ],
+  framework: '@storybook/react',
   core: {
-    builder: 'webpack5',
+    builder: '@storybook/builder-vite',
   },
-  staticDirs: [],
-  webpackFinal: async (config, { configType }) => {
-    // `configType` has a value of 'DEVELOPMENT' or 'PRODUCTION'
-    // You can change the configuration based on that.
-    // 'PRODUCTION' is used when building the static version of storybook.
-
-    const isDevelopment = configType === 'DEVELOPMENT';
-
-    config.module.rules.push({
-      test: /\.s?css$/,
-      use: [
-        'style-loader',
-        {
-          loader: 'css-loader', // turn url() and @import calls into require
-          options: {
-            sourceMap: isDevelopment,
-          },
-        },
-        {
-          loader: 'postcss-loader',
```

Comment:
Given we no longer have this config, check that we still need all the dependencies in the package if you haven't already?